apps/api/resources.prod/api/index.ttl

```diff
@@ -0,0 +1,3 @@
+PREFIX view-builder: <https://view-builder.described.at/>
+
+<> view-builder:endpoint <$METADATA_ENDPOINT> .
```

Comment:
And in production this will be substituted when app starts

(also int but I did not have an idea for a good common name for any deployed environment)

Reply:
Maybe `live` can be used instead?

Reply:
I like that. Done!